wiki/Tournaments/Common_seeding_methods/en.md

```diff
@@ -0,0 +1,181 @@
+
+As always, a third party should be able to get the same results without needing additional information.
+
+## Mini-glossary
+
+In the explanations below, we use the following terms:
+
+- A **team** is a single entity participating in the tournament. For example, the 6–8 players from each country in [OWC](/wiki/Tournaments/OWC) are considered one team, while each player in the [Lazer Grand Arena](/wiki/Tournaments/LGA/2024) is considered their own team.
+- A **score** is a numerical value used for determining a team's performance on a given map. For example, in a 3v3 team tournament, a score typically refers to the sum of the three team members' scores, rather than those three numbers separately. Some tournaments may allow teams multiple runs through a Qualifier mappool; in those cases, the team's score is often either the highest or the average of those runs.
+- A team's **seed** is a number indicating their overall ranking among all teams. Typically, after one of the seeding methods below is used, the top N teams (that is, the teams seeded 1, 2, ..., N) are placed into an elimination bracket or other competition format, with their locations being determined by their seeds.
+- **Normalisation** is the process of dividing a team's score by any mod multipliers on the map. Typically, this is done to avoid artificial inflation of scores on modded maps.
+
+## Seeding methods
+
+<!-- NOTE FOR WIKI EDITORS: Please do not add or remove seeding methods without prior approval of the Tournament Committee -->
+
+Each subsection below explains, both in words and in a mathematical formula, a different seeding method commonly found in osu! tournaments. In all formulas, `m` refers to the number of maps in the qualifier mappool.
+
+### Average Score
+
+This seeding method averages each team's scores on all maps. Typically, scores are first normalised. Teams are then seeded in order of this average, with the highest average receiving seed 1.
+
+The average score for a team is given by
+
+![Average Score formula](img/avgscore.png)
+
+If any of the maps are played with [Free Mod](/wiki/Gameplay/Game_modifier#free-mod), it must be explicitly specified whether each individual player's score on the map is normalised by the mods they use — effectively nullifying multipliers of any additional mods taken — or if there is an overall normalisation — where all scores are divided only by the globally required mod multipliers.
+
+### Sum of Placements
```

Comment:
we generally don't randomly declare proper nouns unless it pertain to spelling used in the game (e.g. mod names). other articles instead use bolding to introduce new terms

- Ziph's law (normally, only "Ziph" is the proper noun since it's a surname)
- Z-score (letters by themselves should be capitalised in any case)
- Z-sum
- average score
- sum of placements
- percent max
- percent difference